src/renderer/corporate/member/corporate-add-member-container.jsx

```diff
@@ -112,12 +199,13 @@ class CorporateAddMemberContainer extends PureComponent {
 	}
 
 	render() {
-		const membersForm = _.pick(this.state, 'errors');
+		const membersForm = _.pick(this.state, 'errors', this.selectFields(this.state.type));
```

Comment:
does it work? as far as I know `_.pick` requires an array as second argument

Reply:
<img width="240" alt="Screenshot 2019-11-19 at 16 35 55" src="https://user-images.githubusercontent.com/1074/69167516-04117000-0aed-11ea-99a7-86c55bc7504f.png">
Yikes! It works, that's why I didn't catch it. 
But it's undocumented, so I fixed it on https://github.com/SelfKeyFoundation/Identity-Wallet/pull/1832/commits/9a6b064081cc76426423b17776b0bb8e4499c16e